demo/test.c

```diff
@@ -223,9 +223,9 @@ static int test_mp_get_set_i64(void)
 
    DOR(mp_init(&a));
 
-   check_get_set_i64(&a, 0LL);
```

Comment:
Here the cast is unnecessary. I would prefer if we only add casts where needed, e.g., in code like `(uint64_t)1 << ...`

Reply:
Agreed. Just as the "LL" postfix shouldn't have been here originally in the first place ;-)

Reply:
Yes, unfortunately fperrad is overdoing it. You've seen me probably complaining about the linting before ;)